src/components/Projects/hooks/useInfiniteProjectsScroll.ts

```diff
@@ -7,10 +7,19 @@ import {
 const useInfiniteProjectsScroll = ( { params: newInputParams, enabled }: object ): object => {
   const baseParams = {
     ...newInputParams,
-    per_page: 50,
+    per_page: 20,
     ttl: -1,
     rule_details: true,
-    fields: "all"
+    fields: {
```

Comment:
Does it make sense to turn these field sets into constants, similar to `Observation.ADVANCED_MODE_LIST_FIELDS` that you created for observation fetching.